### PR TITLE
UUID creation

### PR DESCRIPTION
**Pull request by cchulo**

### CODE REVIEW

File: utils/uuidgen.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const uuid = require('uuid/v4');
+const _ = require('underscore');
+
+// store uuids here
+uuidList = [];
```
Comment (KennyWibowo) [inline utils/uuidgen.js]: why not a set? that way its O(1) lookup instead of O(n)

File: routes/index.js [hunk 2]
```diff
@@ -356,6 +357,19 @@ router.post('/register-admin', (req, res) => {
...
     res.status(500).json("access denied");
   }
-})
+});
+
+/* GET uuid-gen - generate a uuid, store it in list, send it back to user */
+router.get('/uuid-gen', (req, res) => {
+  var uuidStr = uuid.GenerateUUID();
+  uuid.addUUIDToList(uuidStr);
+  req.flash('success', uuidStr);
```
Comment (KennyWibowo) [inline routes/index.js]: Maybe flash the resultant URL instead of just the UUID?

File: routes/index.js [hunk 3]
```diff
@@ -356,6 +357,19 @@ router.post('/register-admin', (req, res) => {
...
+  res.redirect('/new-job');
+});
+
+/* GET new-job - visit new job page */
+router.get('/new-job', (req, res) => {
```
Comment (KennyWibowo) [inline routes/index.js:372]: How is the UUID used in `/new-job`?

**Review by KennyWibowo — CHANGES_REQUESTED** [review]
Good work so far; some changes needed

File: routes/index.js (hunk 3 above)
Comment (cchulo) [reply routes/index.js:372]: the page new-job has a button to generate uuid, returned in a flash when it gets from /uuid-gen (which will redirect to /new-job)

File: routes/index.js (hunk 2 above)
Comment (cchulo) [reply routes/index.js]: i will do that until the url has been decided

File: routes/index.js (hunk 3 above)
Comment (KennyWibowo) [reply routes/index.js:372]: Oo i see, so this is the new job creation page

**Review by KennyWibowo — APPROVED** [review]
Looks good to me!

**Pull request merged**